2011-03-01  Aurelien Jarno  <aurelien@aurel32.net>

	* Makefile (tst-_res1): Add tst-_res1mod1 to dependency list.

---
 linuxthreads/Makefile |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/linuxthreads/Makefile
+++ b/linuxthreads/Makefile
@@ -314,7 +314,8 @@
 
 ifeq (yes,$(build-shared))
 $(objpfx)tst-_res1mod2.so: $(objpfx)tst-_res1mod1.so
-$(objpfx)tst-_res1: $(objpfx)tst-_res1mod2.so $(shared-thread-library)
+$(objpfx)tst-_res1: $(objpfx)tst-_res1mod1.so $(objpfx)tst-_res1mod2.so \
+		    $(shared-thread-library)
 
 $(objpfx)tst-tls1: $(objpfx)tst-tls1mod.so $(shared-thread-library)
 
